src/Module/BaseApi.php

```diff
@@ -142,25 +142,28 @@ public static function unsupported(string $method = 'all')
 	 * @return array request data
 	 */
 	public static function getRequest(array $defaults) {
```

Comment:
Code standards: Please add a new line before method opening braces.
```suggestion
	public static function getRequest(array $defaults)
	{
```